doc/source/wsrep-system-index.rst

```diff
@@ -603,7 +595,6 @@ The following methods are available:
 
 .. variable:: wsrep_preordered
 
-   :version 5.7.24-31.33: Variable deprecated
    :cli: ``--wsrep-preordered``
```

Comment:
Variable has been removed in 8.0, not sure if we need the description. Same goes for other previously deprecated variables that are now removed.